triton-server/src/download/actions.ts

```diff
@@ -85,10 +86,7 @@ export async function createActions(db: Kysely<Database>) {
                 .select("expiry_days")
                 .executeTakeFirstOrThrow()
         ).expiry_days
-        const now = new Date()
-        const expiryDate = new Date(
-            now.setDate(now.getDate() + extensionSize),
-        ).toISOString()
+        const expiryDate = dayjs().add(extensionSize, "days").format()
```

Comment:
the js language default object Date can do the same work, no need to have an extra package to handle the date